### PR TITLE
Mat 6396 value set details

### PR DESCRIPTION
**Pull request by gregory-akins**
## MADiE PR

Jira Ticket: [MAT-6396](https://jira.cms.gov/browse/MAT-6396)
(Optional) Related Tickets:

### All Submissions
* [X] This PR has the JIRA linked.
* [X] Required tests are included
* [X] No extemporaneous files are included (i.e Complied files or testing results)
* [X] This PR is in to the **correct branch**.
* [X] All Documentation as needed for this PR is Complete (or noted in a TODO or other Ticket)
* [ ] Any breaking changes or failing automation are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package)
* [ ] All CDN/Web dependancies are hosted internally (i.e MADiE-Root Repo)

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
*  The tests appropriately test the new code, including edge cases
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads

### CODE REVIEW

**Review by mcmcphillips — APPROVED** [review]

File: src/terminologySectionPanel/ValueSets/Results/Results.tsx [hunk 1]
```diff
@@ -143,6 +167,21 @@ export default function Results(props: ResultsProps) {
...
+    const terminologyService = await useTerminologyServiceApi();
+    const result = await terminologyService.getValueSet(oid, undefined, true);
+    return JSON.stringify(result, null, 2);
+  };
+
+  const handleDetailsClick = async () => {
+    setOpenPopoverOptions(false);
+    //get teh OID number
+    //urn:oid:2.16.840.1.113762.1.4.1099.53 -> 2.16.840.1.113762.1.4.1099.53
+    const oid = selectedReferenceId.slice(8);
```
Comment (adongare) [inline src/terminologySectionPanel/ValueSets/Results/Results.tsx:180]: I'm not sure if this is going to be true always

**Review by adongare — APPROVED** [review]
